python/pyspark/cloudpickle/compat.py

```diff
@@ -1,5 +1,18 @@
 import sys
 
 
-import pickle  # noqa: F401
-from pickle import Pickler  # noqa: F401
+if sys.version_info < (3, 8):
```

Comment:
I brought this back just to match with the exiting codebase in cloudpickle, cc @dongjoon-hyun FYI